deep_qa/data/instances/sequence_tagging/verb_semantics_instance.py

```diff
@@ -0,0 +1,173 @@
+from typing import Dict, List, Tuple
+
+import numpy
+from overrides import overrides
+
+from ..instance import TextInstance, IndexedInstance
+from ...data_indexer import DataIndexer
+
+
+class VerbSemanticsInstance(TextInstance):
+    """
+    A VerbSemanticsInstance is a :class:`TextInstance` that is a single sentence of text,
+    where that sentence has some associated categorical label.
+    """
+    def __init__(self, sentence: List[str], verb: (int, int), entity: (int, int), state_change: str, arg1: (int, int), arg2: (int, int), index: int=None):
+        """
+        sentence: a sentence from which we want to predict verb semantics.
+        A sentence can mention multiple verbs and entities, a data instance
+        is talking about a particular (verb, entity) pair in a given sentence
+        verb: span of the verb indicating the event being referred to in the query
+        entity: entity indicating 
+        """
+        """ TODO: initialize all members of the class"""
+        self.sentence = sentence
+        self.verb = verb
+        self.entity = entity
+        tags = self.__make_tag_sequence(arg1, arg2)
+        super(VerbSemanticsInstance, self).__init__((state_change, tags), index)
+
+
+    def __str__(self):
+        return 'VerbSemanticsInstance(' + self.sentence + ', ' + str(self.label) + ')'
```

Comment:
Not a huge deal, but this is for debug output.  You might want to also include the verb and entity in here somehow, too.